helm-chart/kuberay-apiserver/templates/service.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
     prometheus.io/path: /metrics
     prometheus.io/scrape: "true"
     prometheus.io/port: "8888"
+  labels:
```

Comment:
Can you apply the same change to [apiserver.yaml](./apiserver/deploy/base/apiserver.yaml)?

Reply:
Done